web/src/js/ducks/view.js

```diff
@@ -0,0 +1,36 @@
+import ReduxThunk from 'redux-thunk'
+
+import { SELECT_FLOW } from './flows'
+const ACTIVE_MENU = 'ACTIVE_MENU'
+
+
+const defaultState = {
+    active_menu: 'Start'
+}
+export default function reducer(state = defaultState, action) {
+    switch (action.type) {
+        case ACTIVE_MENU:
+            return {
+                ...state,
+                active_menu: action.active_menu
+            }
+        case SELECT_FLOW:
+            if (!action.currentSelection != !action.flowId){
```

Comment:
Nevermind - didn't see the `!`. We can make this code more clear by introducing variables for the conditions that describe our intent, e.g.  
```
let isNewSelection = (action.flowId && !action.currentSelection)
if(isNewSelection){
  // ...
}
```

Besides, we don't have a DESELECT event yet. I'm wondering if it's nicer to have `SELECT_FLOW` with `flowId=undefined` or a dedicated event for that. I'll leave that up to you. :smiley: